group_project_v2/stage/base.py

```diff
@@ -249,14 +266,41 @@ def get_dashboard_stage_state(self):
 
         return stage_state, state_stats
 
-    def get_stage_stats(self):  # pylint: disable=no-self-use
-        # TODO - real stats calculation
+    def get_stage_stats(self, target_workgroups, target_users):  # pylint: disable=no-self-use
+        target_user_ids = set(user.id for user in target_users)
+        if not target_user_ids:
+            return {
+                StageState.COMPLETED: 0,
+                StageState.INCOMPLETE: 0,
+                StageState.NOT_STARTED: 0
+            }
+
+        target_user_count = len(target_user_ids) * 1.0
```

Comment:
Nit: Use `float(...)` rather than multiplication if you want to cast to a float.

Or, even better, just put `from __future__ import division` at the top of the file.